createARBfile.py

```diff
@@ -0,0 +1,308 @@
+import json
```

Comment:
What is the purpose of this file?

Reply:
It lets me(us) use Google Gemini to automatically convert a sentence in hebrew to he and en arb files format. 
It's used outsize of the app itself, but has value for the project.